docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md

```diff
@@ -142,7 +142,7 @@ export default class ListItems extends React.Component<IListItemsProps, {}> {
                 Welcome to SharePoint!
               </span>
               <p className='ms-font-l ms-fontColor-white'>
-                Customize SharePoint experiences using Web Parts.
+                Customize SharePoint experiences using web parts.
```

Comment:
This is the standard string that is created by the SharePoint Framework Yeoman generator.